source/standards/vulnerability-disclosure.html.md.erb

```diff
@@ -65,7 +65,7 @@ researcher, check with them first and ask which name they wish to have
 displayed.
 
 
-[CDIO Cyber Security team]: https://gds.slack.com/messages/CCMPJKFDK/
+[CDIO Cyber Security team]: https://sites.google.com/d/1xS62XUT7JhWByefe_WeSz1pAVpx3KnTM/p/1uu0YzttUQja-cOZ9ZDHSIqRbd75A88H6/
```

Comment:
That link doesn't work for me - should it be https://sites.google.com/cabinetoffice.gov.uk/cybersecurity ?